ui-tests/src/test/java/org/openmrs/reference/page/RegistrationEditSectionPage.java

```diff
@@ -23,6 +23,7 @@
     private static final By STATE_PROVINCE = By.id("stateProvince");
     private static final By COUNTRY = By.id("country");
     private static final By POSTAL_CODE = By.id("postalCode");
+    private static final String STR = "Must be a valid phone number (with +, -, numbers or parentheses))";
```

Comment:
@sherrif10 what do you think about getting a string name more readable than `STR` ?

Reply:
Nice catch. thanks Resolved